samples/cpp/benchmark_app/README.md

```diff
@@ -188,7 +188,7 @@ Running the application with the empty list of options yields the usage message
 The benchmark tool supports topologies with one or more inputs. If a topology is not data sensitive, you can skip the input parameter, and the inputs will be filled with random values. If a model has only image input(s), provide a folder with images or a path to an image as input. If a model has some specific input(s) (besides images), please prepare a binary file(s) that is filled with data of appropriate precision and provide a path to it as input. If a model has mixed input types, the input folder should contain all required files. Image inputs are filled with image files one by one. Binary inputs are filled with binary inputs one by one.
 
 ## Examples of Running the Tool
-This section provides step-by-step instructions on how to run the Benchmark Tool with the `asl-recognition` Intel model on CPU or GPU devices. It uses random data as the input.
+This section provides step-by-step instructions on how to run the Benchmark Tool with the `googlenet-v1` model from the Open Model Zoo on CPU or GPU devices. It uses random data as the input.
```

Comment:
Why is this changed to a model that needs omz_converter? I liked that this was a minimal example

Reply:
Previously, I wasn't able to get the `asl-recognition-0004` model working correctly with Dynamic Shapes in benchmark_app. I thought maybe it was because `asl-recognition-0004` wasn't compatible with Dynamic Shapes, so I reverted back to using `googlenet-v1` for the example.

However, I later realize that I just needed to add the `-pcseq` command to get benchmark_app to make it correctly show output results for each Dynamic Shape. Now it works with `asl-recognition-0004` as expected. Now that I have it working, I'll change back to using the `asl-recognition-0004` model for the example, which doesn't require omz_converter.

(See PR #13053 for a history)

Reply:
The examples on the Python and C++ pages have been changed to use `asl-recognition-0004`.